src/main/kotlin/org/rust/lang/core/psi/ext/RsAttr.kt

```diff
@@ -20,3 +22,11 @@ val RsAttr.owner: RsDocAndAttributeOwner?
         // Throw exception so that any problems with this property aren't silent and are instead easily findable
         else -> error("Unsupported attribute type: $this")
     }
+
+val RsAttr.isBuiltInAttribute: Boolean
```

Comment:
I think we should rename it somehow because in case of collision it's still builtin attribute

Reply:
do you mean `metaItem.name`?